features.bzl

```diff
@@ -16,6 +16,15 @@ _external_deps = struct(
     is_bzlmod_enabled = str(Label("//:invalid")).startswith("@@"),
 )
 
+_flags = struct(
+    # This flag was renamed in https://github.com/bazelbuild/bazel/pull/18313
+    allow_unresolved_symlinks = (
+        "allow_unresolved_symlinks" 
+        if ge("7.0.0")
```

Comment:
@Wyverald Do Bazel prereleases return their full version string?

If so, we may want to make this [7.0.0-pre.20230628.2](https://github.com/bazelbuild/bazel/releases/tag/7.0.0-pre.20230628.2) instead.

Reply:
```
$ bazel --version
2023/07/27 18:01:42 Downloading https://releases.bazel.build/7.0.0/rolling/7.0.0-pre.20230628.2/bazel-7.0.0-pre.20230628.2-darwin-arm64...
bazel 7.0.0-pre.20230628.2
```

Yes

Reply:
fixed, and tested:

```
alexeagle@system76-pc:~/Projects/rules_js$ USE_BAZEL_VERSION=7.0.0-pre.20230628.2 bazel build --nobuild js/
...  --override_repository=bazel_features=$HOME/Projects/bazel_features
2023/07/27 09:15:49 Downloading https://releases.bazel.build/7.0.0/rolling/7.0.0-pre.20230628.2/bazel-7.0.0-pre.20230628.2-linux-x86_64...
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
INFO: Analyzed 224 targets (243 packages loaded, 11227 targets configured).
INFO: Found 224 targets...
INFO: Elapsed time: 9.067s, Critical Path: 0.00s
INFO: 0 processes.
INFO: Build completed successfully, 0 total actions
INFO: Build Event Protocol files produced successfully.
alexeagle@system76-pc:~/Projects/rules_js$ USE_BAZEL_VERSION=7.0.0-pre.20230530.3 bazel build --nobuild js/
...  --override_repository=bazel_features=$HOME/Projects/bazel_features
2023/07/27 09:16:33 Downloading https://releases.bazel.build/7.0.0/rolling/7.0.0-pre.20230530.3/bazel-7.0.0-pre.20230530.3-linux-x86_64...
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
INFO: Analyzed 224 targets (241 packages loaded, 11217 targets configured).
INFO: Found 224 targets...
INFO: Elapsed time: 10.146s, Critical Path: 0.00s
INFO: 0 processes.
INFO: Build completed successfully, 0 total actions
INFO: Build Event Protocol files produced successfully.
```